presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -188,6 +189,31 @@ public void shutdown()
         deletionExecutor.shutdownNow();
     }
 
+    private static TypeSignature toStorageType(TypeSignature typeSignature)
+    {
+        List<TypeSignatureParameter> typeSignatureParameterList = typeSignature.getParameters();
```

Comment:
Name the local variable `parameters`. No need for such a long variable when there's only one.